CocoaSchool/HW2.playground/Contents.swift

```diff
@@ -1,23 +1,43 @@
-class Stack {
-    private var array: [Int] = []
+/// This class basic on principle LIFO.
+class Stack<Element> {
+    private var array: [Element] = []
     
-    func push(element: Int) {
+    /**
+    Adds element on the top of the stack.
```

Comment:
Use default indentation:
```suggestion
     Adds element on the top of the stack.
```

Reply:
corrected.